ios/RNDidomi.swift

```diff
@@ -17,9 +17,9 @@ class RNDidomi: RCTEventEmitter {
     override static func requiresMainQueueSetup() -> Bool {
         return true
     }
-    
-    @objc(initialize:userAgentVersion:apiKey:localConfigurationPath:remoteConfigurationURL:providerId:disableDidomiRemoteConfig:languageCode:noticeId:)
-    func initialize(userAgentName: String, userAgentVersion: String, apiKey: String, localConfigurationPath: String?, remoteConfigurationURL: String?, providerId: String?, disableDidomiRemoteConfig: Bool = false, languageCode: String? = nil, noticeId: String? = nil) {
+
+    @objc(initialize:userAgentVersion:apiKey:localConfigurationPath:remoteConfigurationURL:providerId:disableDidomiRemoteConfig:languageCode:noticeId:resolve:reject:)
+    func initialize(userAgentName: String, userAgentVersion: String, apiKey: String, localConfigurationPath: String?, remoteConfigurationURL: String?, providerId: String?, disableDidomiRemoteConfig: Bool = false, languageCode: String? = nil, noticeId: String? = nil, resolve:RCTPromiseResolveBlock,reject:RCTPromiseRejectBlock) {
```

Comment:
there seems to be a missing space between the last 2 parameters

Reply:
Fixed